package.json

```diff
@@ -14,24 +15,34 @@
     "bitcoinjs"
   ],
   "scripts": {
-    "coverage-report": "nyc report --reporter=lcov",
-    "coverage-html": "nyc report --reporter=html",
-    "coverage": "nyc --check-coverage --branches 90 --functions 90 mocha",
-    "integration": "mocha --timeout 50000 test/integration/",
-    "standard": "standard",
-    "test": "npm run standard && npm run coverage",
-    "unit": "mocha"
+    "build": "tsc -p ./tsconfig.json",
+    "coverage-report": "npm run build && npm run nobuild:coverage-report",
+    "coverage-html": "npm run build && npm run nobuild:coverage-html",
+    "coverage": "npm run build && npm run nobuild:coverage",
+    "integration": "npm run build && npm run nobuild:integration",
+    "nobuild:coverage-report": "nyc report --reporter=lcov",
+    "nobuild:coverage-html": "nyc report --reporter=html",
+    "nobuild:coverage": "nyc --check-coverage --branches 90 --functions 90 --lines 90 mocha",
+    "nobuild:integration": "mocha --timeout 50000 test/integration/",
+    "nobuild:standard": "standard ts_src/**/*.ts",
+    "nobuild:unit": "mocha",
+    "prepare": "npm run build",
+    "standard": "npm run build && npm run nobuild:standard",
+    "test": "npm run build && npm run nobuild:standard && npm run nobuild:coverage",
+    "unit": "npm run build && npm run nobuild:unit"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/bitcoinjs/bitcoinjs-lib.git"
   },
   "files": [
-    "src"
+    "src",
+    "types"
   ],
   "dependencies": {
+    "@types/node": "10.12.18",
```

Comment:
I _prefer_ `@types/node` to be included in devDependencies, or you might have the solid reason to add it to dependencies?

Reply:
From my understanding by issues like this:
https://stackoverflow.com/a/46011417

1. Applications should set all types to devDependencies.
2. Libraries should set any types that are exposed to the consumer via themselves in dependencies, and devDependencies should only be for types that are solely used in testing.

That being said... I think most apps will have `@types/node`... so maybe it is ok to move to devDependencies.

Reply:
> I think most apps will have @types/node... so maybe it is ok to move to devDependencies.

:+1: